vowpalwabbit/cats_tree.cc

```diff
@@ -192,9 +192,13 @@ void cats_tree::init_node_costs(v_array<cb_class>& ac)
   _cost_star = ac[0].cost / ac[0].probability;
 
   uint32_t node_id = ac[0].action + _binary_tree.internal_node_count() - 1;
+  // stay inside the node boundaries
```

Comment:
Curious why this is needed now and not needed before?

Reply:
This is a guard that we needed before too, for specific combinations of num_actions and bandwidth the tree would return node id's that were out of bounds

Reply:
Got it.  Thanks.